test/unit/spec/ForgotPassword_spec.js

```diff
@@ -350,6 +386,85 @@ function (Q, _, $, OktaAuth, Util, AccountRecoveryForm, PrimaryAuthForm, Beacon,
           });
         });
       });
+      itp('shows an error if username is empty and request Voice Call', function () {
+        return setupWithCall().then(function (test) {
+          $.ajax.calls.reset();
+          test.form.makeCall();
+          expect($.ajax).not.toHaveBeenCalled();
+          expect(test.form.usernameErrorField().length).toBe(1);
+        });
+      });
+      itp('makes a Voice Call', function () {
+        return setupWithCall().then(function (test) {
+          $.ajax.calls.reset();
+          test.setNextResponse(resChallengeCall);
+          test.form.setUsername('foo');
+          test.form.makeCall();
+          return tick();
```

Comment:
I added support for wait functions so we ~mostly~ no longer need to rely on ticks (there are a couple edge cases). Aside from being more stable, it allows us to be explicit about which pages we're waiting for. In this case, we should replace "tick()" with "Expect.waitForRecoveryChallenge()".

There's also examples of this in this file, i.e:
https://github.com/mauriciocastillosilva-okta/okta-signin-widget/blob/884472c3748235490377b1d8b7ba51ed28f0e693/test/unit/spec/ForgotPassword_spec.js#L577